unison-cli/src/Unison/LSP/FileAnalysis.hs

```diff
@@ -272,13 +271,24 @@ getFileAnalysis uri = do
   pure $ Map.lookup uri checkedFiles
 
 -- TODO memoize per file
-ppeForFile :: Uri -> Lsp PrettyPrintEnv
-ppeForFile fileUri = do
-  ppe <- PPE.suffixifiedPPE <$> globalPPE
+ppedForFile :: Uri -> Lsp PPED.PrettyPrintEnvDecl
```

Comment:
I pulled out the meat of this function into a helper, so I can use that helper when generating the file analysis itself.